.gitignore

```diff
@@ -14,3 +14,7 @@
 vendor
 # GoLand IDE
 .idea/
+
+cmd/schemagen/schemagen
+cmd/schemagen/models
+cmd/schemagen/asdf
```

Comment:
Do we need that, can we generate to temp dir.

Reply:
Will just remove it, tests do the clean up